### PR TITLE
enh(comments): Add visual labels for editor

### LINKED ISSUE #41884: [BITV] 9.3.3.2/3.1 - Comment section: The input fields for creating and editing comments lack visual labels. (2)
image

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-9f92015c98-v3-n1

9.3.3.2/11.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-9f92015c98-v11-n1

### PR DESCRIPTION
**Pull request by susnux**
* Resolves: https://github.com/nextcloud/server/issues/41884

## Summary
Add visual labels for the comments input.

Screenshot_20240110_020151
Screenshot_20240110_020311

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/comments/src/components/Comment.vue [hunk 1]
```diff
@@ -76,6 +76,8 @@
 					<NcRichContenteditable ref="editor"
 						:auto-complete="autoComplete"
 						:contenteditable="!loading"
+						:label="editor ? t('comments', 'Write new comment') : t('comments', 'Edit comment')"
+						:placeholder="t('comments', 'Write a comment…')"
```
Comment (ShGKme) [inline apps/comments/src/components/Comment.vue:80]: Not 100% sure, but 90% sure, especially with "write" and a space before `…`.

```suggestion
						:label="editor ? t('comments', 'Write a new comment') : t('comments', 'Edit a comment')"
						:placeholder="t('comments', 'Write a comment …')"
```

Comment (susnux) [reply apps/comments/src/components/Comment.vue]: Done :white_check_mark:

**Review by ShGKme — APPROVED** [review]

**Review by emoral435 — APPROVED** [review]
Tested on my local dev env, looks good to me :) just needs the backport, unfortunately it seems like the backport bot is still snoozing a bit ;/ haha

File: apps/comments/src/components/Comment.vue (hunk 1 above)
Comment (ShGKme) [reply apps/comments/src/components/Comment.vue]: Article is not needed in this case?

**Pull request merged**